aot/src/main/java/com/dylibso/chicory/aot/AotEmitters.java

```diff
@@ -1,17 +1,9 @@
 package com.dylibso.chicory.aot;
 
 import static com.dylibso.chicory.aot.AotMethods.*;
-import static com.dylibso.chicory.aot.AotUtil.boxer;
-import static com.dylibso.chicory.aot.AotUtil.emitInvokeStatic;
-import static com.dylibso.chicory.aot.AotUtil.emitInvokeVirtual;
-import static com.dylibso.chicory.aot.AotUtil.methodNameFor;
-import static com.dylibso.chicory.aot.AotUtil.methodTypeFor;
-import static com.dylibso.chicory.aot.AotUtil.stackSize;
-import static com.dylibso.chicory.aot.AotUtil.unboxer;
-import static com.dylibso.chicory.aot.AotUtil.validateArgumentType;
+import static com.dylibso.chicory.aot.AotUtil.*;
```

Comment:
- unrelated - this shows a problem with the spotless setup, as [this rule](https://github.com/dylibso/chicory/blob/df74f2a258de42e5eebfda4f6e34bc9784477f5d/pom.xml#L388) should prevent star imports.
In this specific case, it reduces the pain of having an `Util` class, which is something I would possibly discourage.

Reply:
oh, maybe the regexp is not capturing `import static`?

Reply:
I didn't realize that star imports were supposed to be disallowed in Chicory. We've always banned them in other projects. I had to change my IntelliJ settings for Chicory to *not* remove the star imports.

Note that `AotMachine` has `OpCode.*` and expanding that will add ~200 imports, since it has one per opcode. IntelliJ hides imports, so I don't care either way, just FYI.

Reply:
I put up a PR to remove wildcard imports and updated this PR to not add a new wildcard import.